src/databases/rocks_db.rs

```diff
@@ -5,7 +5,6 @@ use std::{
     path::Path,
 };
 
-use log::trace;
```

Comment:
This is very useful to debug, is that possible to keep it for std ?

Reply:
we should even be able to keep it for wasm i'll add it back

Reply:
done